4.processing_image_based_profiles/notebooks/2.merge_sc.ipynb

```diff
@@ -0,0 +1,622 @@
+                            "0                                         1.33351   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Difference.Variance_256.1  \\\n",
+                            "0                                         0.002369   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Entropy_256.1  \\\n",
+                            "0                              2.76259   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Information.Measure.of.Correlation.1_256.1  \\\n",
+                            "0                                          -0.507738                  \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Information.Measure.of.Correlation.2_256.1  \\\n",
+                            "0                                           0.920417                  \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Inverse.Difference.Moment_256.1  \\\n",
+                            "0                                           0.831104       \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Sum.Average_256.1  \\\n",
+                            "0                                 8.297708   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Sum.Entropy_256.1  \\\n",
+                            "0                                 2.122714   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Sum.Variance_256.1  \\\n",
+                            "0                                198.636846   \n",
+                            "\n",
+                            "   Texture_Organoid_Mito_Variance_256.1  \n",
+                            "0                              50.32471  \n",
+                            "\n",
+                            "[1 rows x 642 columns]"
+                        ]
+                    },
+                    "execution_count": 8,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "# save the organoid data as parquet\n",
+                "print(f\"Final organoid data shape: {merged_df.shape}\")\n",
+                "organoid_table.to_parquet(destination_organoid_parquet_file, index=False)\n",
+                "organoid_table.head()"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 9,
+            "metadata": {},
+            "outputs": [
+                {
+                    "name": "stdout",
+                    "output_type": "stream",
+                    "text": [
+                        "Final merged dataframe shape: (30, 1926)\n"
```

Comment:
Consider labeling this print in context to the record type (e.g. if not an organoid).